src/electrum/server.rs

```diff
@@ -557,19 +557,22 @@ impl Connection {
                             );
 
                             let reply = self.handle_command(method, params, id)?;
+                            let reply_length = reply.to_string().as_bytes().len();
+
+                            let send_result = self.send_values(&[reply]);
```

Comment:
Why do you want to include the time spent sending the reply? This is what I commented on #75:

> This would make the reported duration depend on the client's connection speed, is this something we want?
>
> Typically the time spent to send the response is proportional to the response size which is already reported, perhaps that's sufficient?